### PR TITLE
[master] Version upgrades

### PR DESCRIPTION
**Pull request by mathieucarbou**
**Upgraded to:**

 * terracotta-utilities = 0.0.10
 * terracottaPlatformVersion = 5.8.14
 * terracottaCoreVersion = 5.8.7
 * jacksonVersion = 2.13.2.20220328

### CODE REVIEW

File: ehcache-xml/build.gradle [hunk 1]
```diff
@@ -129,7 +129,7 @@ tasks.named('jakartaJar', Jar) {
   bundle {
     bnd (
       'Export-Package': 'org.ehcache.xml.*',
-      'Import-Package': "jakarta.xml.bind*;version=\"[3,4)\", *"
+      'Import-Package': "jakarta.xml.bind*;version=\"[3,3.1)\", *"
```
Comment (chrisdennis) [inline ehcache-xml/build.gradle:132]: Should be `[3,4)` since this is exported by the jaxb-api.

File: ehcache/build.gradle [hunk 2]
```diff
@@ -102,7 +102,7 @@ tasks.named('jakartaJar') {
     instruction Constants.BUNDLE_DESCRIPTION, 'Ehcache is an open-source caching library, compliant with the JSR-107 standard.'
     instruction Constants.BUNDLE_ACTIVATOR, 'org.ehcache.core.osgi.EhcacheActivator'
     instruction Constants.EXPORT_PACKAGE, '!org.ehcache.jsr107.tck, !org.ehcache.*.internal.*, org.ehcache.*'
-    instruction Constants.IMPORT_PACKAGE, 'javax.cache.*;resolution:=optional, !javax.annotation, !sun.misc, jakarta.xml.bind*;version="[3,4)", *'
+    instruction Constants.IMPORT_PACKAGE, 'javax.cache.*;resolution:=optional, !javax.annotation, !sun.misc, jakarta.xml.bind*;version="[3,3.1)", *'
```
Comment (chrisdennis) [inline ehcache/build.gradle:105]: Should be [3,4) since this is exported by the jaxb-api.

**Review by chrisdennis — CHANGES_REQUESTED** [review]

**Comment by mathieucarbou** [comment]
@chrisdennis : a random test failure: [getCache_Missing_Stopped](https://dev.azure.com/TerracottaCI/ehcache/_build/results?buildId=9180&view=ms.vss-test-web.build-test-results-tab&runId=1131248&resultId=100854&paneView=debug)

**Comment by mathieucarbou** [comment]
@chrisdennis : merging now to trigger an EE upgrade PR, otherwise this will be stacked with all other new incoming changes which will complicate things.

This upgrade PR is fine and only contains all the angela fixes, platform fixes and test improvements.

**Review by chrisdennis — APPROVED** [review]

**Pull request merged**